src/items/external-blocks.md

```diff
@@ -85,13 +90,16 @@ There are also some platform-specific ABI strings:
 * `extern "vectorcall"` -- The `vectorcall` ABI -- corresponds to MSVC's
   `__vectorcall` and clang's `__attribute__((vectorcall))`
 
-Finally, there are some rustc-specific ABI strings:
+<<<<<<< HEAD
+=======
+Finally, there are some unstable `rustc`-specific ABI strings:
 
-* `extern "rust-intrinsic"` -- The ABI of rustc intrinsics.
-* `extern "rust-call"` -- The ABI of the Fn::call trait functions.
-* `extern "platform-intrinsic"` -- Specific platform intrinsics -- like, for
-  example, `sqrt` -- have this ABI. You should never have to deal with it.
+* `extern "rust-intrinsic"` -- The ABI of rustc intrinsics, equivalent to `"Rust"`.
+* `extern "rust-call"` -- The ABI of the Fn::call trait functions, equivalent to `"Rust"`.
+* `extern "platform-intrinsic"` -- The ABI of Specific platform intrinsics, equivalent to `"Rust"` -- like, for
```

Comment:
Not sure these should be documented, we've been meaning to remove all 3 for years...
cc @oli-obk at least for intrinsics we could maybe have an intermediary stage where we do `#[rustc_intrinsic] extern "Rust" {...}` instead of `extern "rust-intrinsic" {...}`?

I would maybe call them "pseudo-ABIs", in that they use the ABI string to indicate something special *not purporting to the ABI*, but are otherwise the "Rust" ABI.

Reply:
They shouldn't; see https://github.com/rust-lang-nursery/reference/pull/652#discussion_r314060449.

For the question of moving towards `#[rustc_intrinsic]` please move this to a rust-lang/rust issue.

Reply:
I've removed these :)